configs/config.toml.SAMPLE

```diff
@@ -54,7 +54,17 @@
 [Storage]
   DataDir = "/my/secure/location"
 
+# Uncomment to enable the metrics module which collects and exposes information
+# useful for external monitoring tools. The port on which the `/metrics`
+# endpoint will be available and the frequency with which the metrics will be
+# collected can be customized using the below parameters.
```

Comment:
We are exposing `libp2p_info` in metrics. Given it's a static value available in diagnostics, shouldn't we remove it from metrics? Removing `ExposeLibP2PInfo` from `metrics.go` looks like a low-hanging fruit we can pick in this PR.